icu4c/source/common/ucnvmbcs.cpp

```diff
@@ -2823,7 +2823,7 @@ ucnv_MBCSToUnicodeWithOffsets(UConverterToUnicodeArgs *pArgs,
                  * - If any of the non-initial bytes could be the start of a character,
                  *   we stop the illegal sequence before the first one of those.
                  */
-                UBool isDBCSOnly=(UBool)(cnv->sharedData->mbcs.dbcsOnlyState!=0);
+                bool isDBCSOnly = cnv->sharedData->mbcs.dbcsOnlyState != 0;
```

Comment:
Is this change from UBool to bool intentional, or a remnant of the replacement trick?

Reply:
It doesn't actually matter, but it was definitely accidental, reverted now.